helmfile/50-applications.yaml

```diff
@@ -214,7 +214,7 @@ releases:
     app: prometheus-wc-reader
   chart: ./charts/prometheus-instance
   version: 0.1.0
-  installed: {{ .Values.influxDB.enabled }}
+  installed: false
```

Comment:
I guess that the [thanos-ruler](https://github.com/elastisys/compliantkubernetes-apps/pull/849) pr is interlinked with this pr, but I guess it is OK to disable wc reader here